### DISCUSSION TITLE
Does the Android app not work with .xbel?

### DISCUSSION THREAD

**Discussion by morgannunan** [floccusaddon/floccus#1620] [Q&A]
I have been using the Floccus extension for years, and it works flawlessly with an xbel file I keep in my Nextcloud instance. It never worked great for me with the actual Nextcloud app, and I suspect this is more of a Nextcloud issue than a Floccus one. I just tried using the Nextcloud bookmarks app in my browser and it's really buggy.

Anyway, I installed the Android version and I don't see a way to use .xbel. Is there a way that I'm just not recognizing?

**Comment by marcelklehr** [comment]
Mh, what *do* you see? The WebDAV share option allows syncing via an XBEL or HTML file in a WebDAV share. The Google Drive option allows syncing via an XBEL or HTML file in Google Drive. The git option allows syncing via an XBEL or HTML file in a git repository. :)

**Comment by marcelklehr** [comment]
Also, I'm curious which bugs you've found in the bookmarks app as I maintain that as well :) If you get around to it, I'd be happy about some tickets over at https://github.com/nextcloud/bookmarks :)

**Comment by morgannunan** [comment]
You know what, I was being stupid. I thought I could edit my existing profile and see the WebDAV option. I removed my profile and started creating a new one, and it had all the same options as the desktop extension.
